### ISSUE TITLE
All chapters of mangas from isekaiscan.eu aren't found by Kotatsu

### ISSUE THREAD

**Issue by Zthrowaway** [KotatsuApp/Kotatsu#212]
### Brief summary

Was going to find new manga to read as if I don't already had too many and found that all of the manga from isekaiscan.eu said there were no chapters in Kotatsu but there were in browsers

### Steps to reproduce

1. Search some basic word
2. Scroll down to the bar that says IsekaiScan (eu)
3. Tap on a manga
4. look at the chapters section

### Kotatsu version

3.4.8

### Android version

10

### Device

Oculus Quest 1

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by Zthrowaway** [comment]
Yes the Oculus Quest 1 is a VR headset - but yes it runs apks as apps and is essentially (from my understanding) just a heavily modified version of Android 10
Meta even straight up took the FB app from android devices and put it on the Oculus Store (still with sharp corners too lol)

**Comment by ztimms73** [comment]
@Koitharu transfer this issue to parsers repo, thanks
